src/index.js

```diff
@@ -1,78 +1,226 @@
+import AppBar from './app-bar';
```

Comment:
If we can get the babel 6 in with the state 1 preset, we can use the export extensions to simplifiy this.

All of them can be like:

```js
export  {AppBar} from './app-bar';
```